Karma/Pages/Requests.cshtml.cs

```diff
@@ -37,9 +37,9 @@ public RequestsModel(IRequestRepository requestService, IMessageRepository messa
             MessageService = messageService;
         }
 
-        public void OnGet()
+        public async void OnGet()

```

Comment:
```suggestion
        public async Task<IActionResult> OnGetAsync()
```
Should have a return type, otherwise request page doesn't get request information and crashes. (1)